yarn.lock

```diff
@@ -1808,11 +1808,6 @@ bcrypt-pbkdf@^1.0.0:
   dependencies:
     tweetnacl "^0.14.3"
 
-big.js@^5.2.2:
```

Comment:
Why are there `yarn.lock` changes? We're not changing any dependencies 😕

Reply:
I actually don't know.  I just created a completely new branch off master to start over, and when I dev up the yarn.lock is changed.

Reply:
Ahh, interesting. I was able to re-produce with a clean master with a `dev up`. Makes sense 👍